ush/ufsda/yamltools.py

```diff
@@ -132,6 +136,35 @@ def fv3land_geom_dict(case, levs, ntiles, layout, io_layout):
     }
     return outdict
 
+def fv3aero_geom_dict(case, levs, ntiles, layout, io_layout):
```

Comment:
what is different here besides the fieldmetadata file? I don't think we need the 'time invariant' part, right?

Reply:
I wasn't sure. I can take it out.

Reply:
Yes, the time `invariant part` is land specific.